eth/handler_test.go

```diff
@@ -506,8 +506,12 @@ func testCheckpointChallenge(t *testing.T, syncmode downloader.SyncMode, checkpo
 	var (
 		db      = rawdb.NewMemoryDatabase()
 		config  = new(params.ChainConfig)
-		genesis = (&core.Genesis{Config: config}).MustCommit(db)
+		genesis *types.Block
 	)
+
+	genesis = (&core.Genesis{Config: config}).MustCommit(db)
```

Comment:
... and simply ignore the return value from `MustCommit`, because it's actually what we're doing anyways.